aspnetcore/fundamentals/hosted-services.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Background tasks with hosted services in ASP.NET Core
+author: guardrex
+description: Learn how to implement background tasks with hosted services in ASP.NET Core.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 02/12/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: fundamentals/hosted-services
+---
+# Background tasks with hosted services in ASP.NET Core
+
+By [Luke Latham](https://github.com/guardrex)
+
+In ASP.NET Core, background tasks are implemented as *hosted services*. A hosted service is a class with background task logic that implements the [IHostedService](/dotnet/api/microsoft.extensions.hosting.ihostedservice) interface. This topic provides three hosted service examples:
```

Comment:
background tasks **can be** implemented as hosted services. Is there an article explaining why that is the case? There might also be questions around how these behave when running in IIS (StopAsync will be called when the application pool is being recycled etc).

Reply:
> Is there an article explaining why that is the case? 

Not that I'm aware of.

> There might also be questions around how these behave when running in IIS (StopAsync will be called when the application pool is being recycled etc).

Currently, the comment speaks more to an "unexpected" shutdown and implies that other types of shutdowns are "graceful" and will trigger `StopAsync`. If you have specific things that you think it should call out, I can list them (e.g., app pool recycling ... is it of the "ungraceful" variety?) ... what else should be listed?